pkg/storaged/iscsi-panel.jsx

```diff
@@ -91,16 +89,14 @@
                                             address: _("Unable to reach server")
                                         };
 
-                                    dfd.reject(error);
+                                    reject(error);
                                 });
 
                         progress_callback(null, function () {
                             cancelled = true;
-                            dfd.reject();
+                            reject();
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18487-20230310-143650-f841503b-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)